src/crocoddyl_reaching.cpp

```diff
@@ -37,15 +37,15 @@
 
 namespace panda_torque_mpc
 {
-    CrocoddylReaching::CrocoddylReaching(pin::Model _model_pin, const boost::shared_ptr<pin::GeometryModel>& _collision_model ,
+    CrocoddylReaching::CrocoddylReaching(pin::Model model_pin_, const boost::shared_ptr<pin::GeometryModel>& collision_model_ ,
         CrocoddylConfig _config) :
     config_(_config)
     {
 
-        auto end_effector_frame_id = _model_pin.getFrameId(_config.ee_frame_name);
+        auto end_effector_frame_id = model_pin_.getFrameId(_config.ee_frame_name);
```

Comment:
```suggestion
        const std::size_t end_effector_frame_id = model_pin.getFrameId(_config.ee_frame_name);
```
This value is considered immutable later on in the code, so it can be explicitly immutable